packages/native-federation-typescript/typedoc.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": ["../../typedoc.base.json"],
+    "entryPoints": ["src/index.ts"],
```

Comment:
I think that [these entrypoints](https://github.com/module-federation/universe/blob/main/packages/native-federation-typescript/tsup.config.ts#L9) are missing

Reply:
Added, the types are still basic without more annotations but it's a start :) thank you for reviewing